ampligraph/latent_features/models/EmbeddingModel.py

```diff
@@ -1208,14 +1203,14 @@ def compute_score_corruptions(i,
                     # Execute the dependency
                     with tf.control_dependencies(corr_dependency):
                         emb_corr = tf.squeeze(self._entity_lookup(corr_batch))
-                        if corrupt_side == 's+o' or corrupt_side == 's':
+                        if corrupt_side == 's+o' or corrupt_side == 's' or corrupt_side == 's,o':
                             # compute and store the scores batch wise
                             scores_predict_s_c = self._fn(emb_corr, e_p, e_o)
                             scores_predict_s_corruptions_in = \
                                 scores_predict_s_corruptions_in.scatter(tf.squeeze(corr_batch),
                                                                         tf.squeeze(scores_predict_s_c))
 
-                        if corrupt_side == 's+o' or corrupt_side == 'o':
+                        if corrupt_side == 's+o' or corrupt_side == 'o' or corrupt_side == 's,o':
```

Comment:
Similar to above, this can just be:


   if 'o' in corrupt_side':
       do things